src/bedtools/bedtools_bedtobam/config.vsh.yaml

```diff
@@ -0,0 +1,85 @@
+name: bedtools_bedtobam
+namespace: bedtools
+description: Converts feature records (bed/gff/vcf) to BAM format.
+keywords: [Converts, BED, GFF, VCF, BAM]
+links:
+  documentation: https://bedtools.readthedocs.io/en/latest/content/tools/bedtobam.html
+  repository: https://github.com/arq5x/bedtools2
+references:
+  doi: 10.1093/bioinformatics/btq033
+license: GPL-2.0, MIT
```

Comment:
I put it there because I used the bedtools_getfasta as template and there was the GPL-2.0 license there. But yes. I found only MIT license on GitHub.